build_defs/python.build_defs

```diff
@@ -330,12 +334,6 @@ def python_test(name:str, srcs:list, data:list|dict=[], resources:list=[], deps:
     else:
         data.extend(srcs)
 
-    if looks_like_build_label(interpreter):
```

Comment:
I think this precedes `test_tools`, we can just use that now (which is more correct too because it's attached even if it's not in repo)